code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -21,12 +21,18 @@
 		return
 	attack(mob/M as mob, mob/user as mob, def_zone)
 		if(M == user)
-			M << "\blue You [apply_method] [src]."
+			if(istype(M,/mob/living/carbon/human))
+				var/mob/living/carbon/human/H = M
+				if(H.species.flags & IS_SYNTHETIC)
+					H << "<span class='warning'>You have a monitor for a head, where do you think you're going to put that?</span>"
+					return
+
+			M << "<span class='notify'>You [apply_method] [src].</span>"
 			M.unEquip(src) //icon update
 			if(reagents.total_volume)
 				reagents.reaction(M, apply_type)
-				spawn(5)
-					reagents.trans_to(M, reagents.total_volume)
+				spawn(0)
+					reagents.trans_to_ingest(M, reagents.total_volume)

```

Comment:
i want to say it's because of either consistancy with feeding it to other people or maybe missing sanity checks on trans_to, but i'm not sure